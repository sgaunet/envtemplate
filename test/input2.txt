{{ indent 4 .MULTILINE }}