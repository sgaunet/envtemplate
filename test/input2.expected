    ligne1
    ligne2
    ligne3